src/status_im/contexts/keycard/check/view.cljs

```diff
@@ -3,16 +3,39 @@
             [react-native.core :as rn]
             [status-im.common.events-helper :as events-helper]
             [status-im.common.resources :as resources]
+            [status-im.constants :as constants]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn- keycard-documentation
+  []
+  [quo/documentation-drawers
+   {:title  (i18n/label :t/keycard)
+    :shell? true}
+   [rn/view
```

Comment:
[:<> ?

Reply:
Thank you @flexsurfer for reviewing the PR.


I tried `[:<>` but its not working well, as shown in the [video](https://github.com/user-attachments/assets/2e12a97b-4e13-4c69-bcb1-09472729727a)